packages/eslint-plugin/src/rules/await-thenable.ts

```diff
@@ -0,0 +1,47 @@
+import * as tsutils from 'tsutils';
+import * as ts from 'typescript';
+
+import * as util from '../util';
+
+export default util.createRule({
+  name: 'await-thenable',
+  meta: {
+    docs: {
+      description: 'Disallows awaiting a value that is not a Thenable',
+      category: 'Best Practices',
+      recommended: 'error',
+      tslintName: 'await-thenable',
+    },
+    messages: {
+      await: 'Invalid `await` of a non-Promise (non-"Thenable") value.',
```

Comment:
I'd like to avoid `invalid` because it's valid as a TypeScript code. How about "Unexpected a non-thenable value in await expression."?